packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -37,11 +37,9 @@ export const RichTextEditor: React.VFC<RichTextEditorProps> = props => {
       schema,
       plugins: [
         history(),
-        keymap({
-          ...baseKeymap,
-          ...buildKeymap(schema),
-        }),
-        buildInputRules(),
+        keymap(buildKeymap(schema)),
+        keymap(baseKeymap),
```

Comment:
When these two things were combined into a single `keymap()` — it meant that some things were getting overwritten, e.g. "Enter". The order also matters, the base keymap needed to come last.

Reply:
Interesting, I suppose it doesn't hurt to delineate between what we're pulling in from prose mirror and what we're defining ourselves